### PR TITLE
fix(controllers): CRLF issues

### PR DESCRIPTION
**Pull request by grkek**

### CODE REVIEW

File: src/controllers/tenants.cr [hunk 1]
```diff
@@ -63,7 +65,7 @@ class Tenants < Application
   # returns the limits for the current domain (Host header)
   @[AC::Route::GET("/current_limits")]
   def current_limits : Limits
-    response.headers["X-Delegated"] = (!!current_tenant.delegated).to_s
+    response.headers["X-Delegated"] = (!!current_tenant.delegated).to_s.gsub(/[\r\n]+/, String.new)
```
Comment (stakach) [inline src/controllers/tenants.cr:68]: don't think there could possibly be an issue here
We're calling: `Bool#to_s`

File: src/controllers/bookings.cr [hunk 2]
```diff
@@ -1,3 +1,5 @@
+require "html"
```
Comment (stakach) [inline src/controllers/bookings.cr:1]: I don't think we need this require? Or is it for something else in the file?

File: src/controllers/tenants.cr (hunk 1 above)
Comment (grkek) [reply src/controllers/tenants.cr:68]: Completely agree, thought would be a precaution to do so and now that I think about it you can not really manipulate the bool out of true or false.

File: src/controllers/bookings.cr (hunk 2 above)
Comment (grkek) [reply src/controllers/bookings.cr:1]: Sorry forgot to remove the require

**Comment by stakach** [comment]
won't fix as headers are protected by the HTTP::Headers library in crystal lang
image

**Pull request closed without merging**